README.md

```diff
@@ -21,22 +21,39 @@ People interested in participating in TAPS can [join the mailing list](https://w
 ## Build Dependencies:
 
 Yang support relies on some shared libraries.  Run the script to download, build,
-and install them into dependencies/install/lib.
+and install them (if not in the default location, then in a place where
+LD_LIBRARY_PATH points).
 
 Requirements:
 
 - gcc or clang
 - cmake
 - libtool
 - autotools
+- libpcre
 
-	./build_dependencies.sh
+~~~
+sudo apt-get update
+sudo apt-get install -y libpcre3-dev cmake
+sudo apt-get install -y autoconf automake libtool
+
+INSTALL_PATH=${HOME}/local_install \
+  ./build_dependencies.sh
+
+
+python3 -m venv ~/venv-taps
+source ~/venv-taps/bin/activate
+
+INSTALL_PATH=${HOME}/local_install \
+  python setup.py build install
```

Comment:
I wrote it this way because it's what I have to actually type. I think on the default installs for both the latest ubuntu and the latest macos, "python" still invokes python 2 unless you're inside a venv of some sort, so I thought a quick mention of how that happens might save someone a bit of time when they're willing to give it a try.  To me that sounds better than consistency, but I'm not opposed to removing the venv line and adding Python3.7+ to the list of requirements, so I did that.